### PR TITLE
[CI:DOCS] Quadlet: clarify overriding user/system services

### PR DESCRIPTION
**Pull request by vrothberg**
Highlight that existing user and system services may be overridden if they have same name as the Quadlet file.  Also point the user to systemctl and how to list existing files.

Closes: #18275

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ corresponding regular systemd service unit files. Both system and user systemd u
 
 The Podman generator reads the search paths above and reads files with the extensions `.container`
 `.volume` and `*.kube`, and for each file generates a similarly named `.service` file. These units
-can be started and managed with systemctl like any other systemd service.
+can be started and managed with systemctl like any other systemd service.  Be aware that existing
```
Comment (ygalblum) [inline docs/source/markdown/podman-systemd.unit.5.md:30]: You have a redundant space after the period. Consider just starting a new line. In addition, maybe this comment should come one sentence earlier, just after we explain that a corresponding `.service` file will be created

Comment (vrothberg) [reply docs/source/markdown/podman-systemd.unit.5.md]: I've shuffled the text a bit.  The double space in between sentences is an old habit when writing monospace text.  I find it much easier to read.

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ These files are read during boot (and when `systemctl daemon-reload` is run) and
 corresponding regular systemd service unit files. Both system and user systemd units are supported.
 
 The Podman generator reads the search paths above and reads files with the extensions `.container`
-`.volume` and `*.kube`, and for each file generates a similarly named `.service` file. These units
-can be started and managed with systemctl like any other systemd service.
+`.volume` and `*.kube`, and for each file generates a similarly named `.service` file. Be aware that
+existing user and system services are replaced if they have the same name. The generated unit files can
```
Comment (Luap99) [inline docs/source/markdown/podman-systemd.unit.5.md:30]: This is not quite true, see https://github.com/containers/podman/issues/18275#issuecomment-1519685487.
The current `normal-dir` only causes us to overwrite "vendor" services (`/usr/...`) but the ones from `/etc/...`

Comment (vrothberg) [reply docs/source/markdown/podman-systemd.unit.5.md]: Really nice catch.  I've updated the text. PTanotherL

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by rhatdan** [comment]
LGTM.  @vrothberg I thought only old farts who leaned to type on typewriters did the double space thing.  I was forced to fix hundreds of these when I was writing the book.  Notice I did double spaces after each period.
/lgtm

**Comment by vrothberg** [comment]
> LGTM. @vrothberg I thought only old farts who leaned to type on typewriters did the double space thing. I was forced to fix hundreds of these when I was writing the book. Notice I did double spaces after each period. /lgtm

I learned on a typewriter :angel: ... and probably am an old fart at heart

**Pull request merged**